js/application.js

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
removed indentation from first line. Indented second line. No indent for closing tag.